.github/copilot-instructions.md

```diff
@@ -0,0 +1,214 @@
+# Rails 7 Starter App - Copilot Instructions
+
+This is a Ruby on Rails 7.2.2.2 web application with Bootstrap 5, Font Awesome, PostgreSQL, and comprehensive tooling for linting, testing, and deployment to Heroku/Render.
```

Comment:
use only the major version of Rails. This application is supposed to test last libraries against a basic rails application at version 7